src/config/production.config.ts

```diff
@@ -22,7 +22,8 @@ const config:IAppConfig = {
     skinRightNavShowExamples: (window as any).skinRightNavShowExamples, 
     skinRightNavShowTestimonials: (window as any).skinRightNavShowTestimonials,
     skinRightNavExamplesHTML: (window as any).skinRightNavExamplesHTML,
-    skinRightNavWhatsNewBlurb: (window as any).skinRightNavWhatsNewBlurb
+    skinRightNavWhatsNewBlurb: (window as any).skinRightNavWhatsNewBlurb,
+    querySetsOfGenes: (window as any).querySetsOfGenes
```

Comment:
There's also a `development.config.ts` you should update. It's a little wonky but these are basically the same files. We should prolly change it to just read one config file.

Reply:
I've updated the `development.config.ts`, too.